src/data/ua_overrides.js

```diff
@@ -47,7 +47,7 @@ const AVAILABLE_UA_OVERRIDES = [
     domain: "att.tv",
```

Comment:
Nit: This isn't needed anywhere but the `about:compat` listing, but for reference, we should probably change the domain here as well. Feel free to pick one (but `directtv.com` is probably more meaningful?)